example.env

```diff
@@ -1,3 +1,4 @@
 MONGO_URI_DEV=mongodb://localhost:27017/review-service-dev
 MONGO_URI_PROD=mongodb://localhost:27017/review-service-prod
-MONGO_URI_TEST=mongodb://localhost:27017/review-service-test
\ No newline at end of file
+MONGO_URI_TEST=mongodb://localhost:27017/review-service-test
+PORT=
```

Comment:
this PORT needs to be filled out for server to properly connect, thinking you might want to fill this in with `5002`?

Reply:
Will add it